### PR TITLE
UX: cleanups of some messages.

### PR DESCRIPTION
**Pull request by concretevitamin**

### CODE REVIEW

File: sky/clouds/azure.py [hunk 1]
```diff
@@ -265,11 +265,11 @@ def check_credentials(self) -> Tuple[bool, Optional[str]]:
...
             return False, (
-                'Azure CLI returned error. Run the following commands '
-                'under sky folder:'
-                # TODO(zhwu): after we publish sky to PyPI, change this to
-                # `pip install sky[azure]`
-                '\n      $ pip install .[azure]'
+                'Azure CLI returned error. Run the following commands:'
+                '\n      $ pip install skypilot[azure]  # if installed from '
+                'PyPI'
+                '\n    Or:'
```
Comment (michaelzhiluo) [inline sky/clouds/azure.py:271]: With the release of wheel, think `pip install .` can be removed. (only relevant for developers)

File: sky/backends/cloud_vm_ray_backend.py [hunk 2]
```diff
@@ -72,10 +72,10 @@
 _WORKER_IP_MAX_ATTEMPTS = 5
 
 _TEARDOWN_FAILURE_MESSAGE = (
-    f'{colorama.Fore.RED}Failed to terminate '
+    f'\n{colorama.Fore.RED}Failed to terminate '
     '{cluster_name}. {extra_reason}'
     'If you want to ignore this error and remove the cluster '
-    'from from Sky\'s status table, use `sky down --purge`.'
+    'from the status table, use `sky down --purge`.'
```
Comment (michaelzhiluo) [inline sky/backends/cloud_vm_ray_backend.py:78]: Run `sky down --purge` to bypass/ignore errors when removing the cluster.

**Review by michaelzhiluo — APPROVED** [review]
LGTM!

File: sky/clouds/azure.py (hunk 1 above)
Comment (concretevitamin) [reply sky/clouds/azure.py:271]: We still include in Install docs that you can build from source. Some users do so.

File: sky/backends/cloud_vm_ray_backend.py (hunk 2 above)
Comment (concretevitamin) [reply sky/backends/cloud_vm_ray_backend.py:78]: The current wording puts the purpose first and action next, which reads clearer to me.

**Pull request merged**